auth/postgres/tracing.go

```diff
@@ -21,6 +21,12 @@ type database struct {
 type Database interface {
 	NamedExecContext(context.Context, string, interface{}) (sql.Result, error)
 	QueryRowxContext(context.Context, string, ...interface{}) *sqlx.Row
+	QueryxContext(context.Context, string, ...interface{}) (*sqlx.Rows, error)
+	// NamedExecContext(context.Context, string, interface{}) (sql.Result, error)
```

Comment:
Please remove unused methods, both from the interface and from the implementation.